.github/workflows/withAptitude.yml

```diff
@@ -0,0 +1,110 @@
+# Not cross-platform, but works
+
+name: withAptitude
+
+# Run this workflow manually from the Actions tab
+on: workflow_dispatch
+
+# A workflow run is made up of one or more jobs that can run sequentially or in parallel
+jobs:
+  # This workflow contains a single job called "populate"
+  populate:
+    # Run on a single platform or use a matrix to test against several
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-latest]
+        python-version: [ '3.8'  ]
+    env:
+       TMPTMPDIR: /tmp
+       OS_CHOICE: ${{ matrix.os }}
+       
+    # Steps represent a sequence of tasks that will be executed as part of the job
+    name: gr-bokeh on ${{ matrix.os }} with python ${{ matrix.python-version }} 
+    steps:
+      - name: Set up Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: ${{ matrix.python-version }}
+          architecture: x64
+       
+      - name: Make a place to store logs
+        run:  mkdir -p ~/artifacts
+
+      - name: Security first
+        run: |
+          mkdir ~/logs
+          sudo apt update
+          sudo apt install -y software-properties-common apt-utils 
+          sudo apt install -y unattended-upgrades
+          sudo unattended-upgrade -d >~/artifacts/securityFirst.log
+
+      - name: Ensure third-party apt support is installed
+        run: |
+          if [ "${OS_CHOICE}" = "ubuntu-latest" ]; then
+             sudo add-apt-repository -y ppa:gnuradio/gnuradio-releases-3.9
+          fi
+          sudo apt update
+
+      - name: Install the tangled web 
+        run: |
+          sudo apt install -qy build-essential npm nodejs cmake cmake-curses-gui ninja-build python3-pip
+          sudo apt install -qy lrzip cmake g++ libboost-all-dev libgmp-dev swig python3-numpy \
+                           python3-mako python3-sphinx python3-lxml doxygen libfftw3-dev \
+                           libsdl1.2-dev libgsl-dev libqwt-qt5-dev libqt5opengl5-dev python3-pyqt5 \
+                           liblog4cpp5-dev libzmq3-dev python3-yaml python3-click python3-click-plugins \
+                           python3-zmq python3-scipy python3-gi python3-gi-cairo gir1.2-gtk-3.0 \
+                           libcodec2-dev libgsm1-dev
+          sudo apt install -qy pybind11-dev python3-matplotlib libsndfile1-dev \
+                           python3-pip libsoapysdr-dev soapysdr-tools
+          pip install pygccxml
+          pip install pyqtgraph
+          sudo apt install -qy libiio-dev libad9361-dev libspdlog-dev python3-packaging
```

Comment:
I don't think there is a need to install the dependencies to run a Pluto. But it shouldn't hurt.